osm2gtfs/core/cache.py

```diff
@@ -50,7 +50,7 @@ def write_file(name, content):
         if not os.path.isdir('data'):
             os.mkdir('data')
         with open(os.path.join('data', name), 'wb') as f:
-            f.write(content.read())
+            f.write(content)
```

Comment:
Yes, we just have to coordinate which one to merge in first. I would be happy to give the preference to #120, as this one is smaller and easier to review first and then rebase here.

Reply:
I agree doing this once is enough but in configuration.py there is a difference between both Pull Requests (in [line 86](https://github.com/grote/osm2gtfs/pull/99/files#diff-07e4c70d96a2beb51112609727f78897R86) of this PR the `read()` method should be deleted). The reason why I created the "PR" on master instead to directly in mapanica-master is because this issue is a bug in currently master branch code.

Reply:
> in configuration.py there is a difference between both Pull Requests (in line 86 of this PR the read() method should be deleted).

@jamescr but doesn't your PR add the `read()` method there as well? I don't see the difference!?

Reply:
My PR added the `read()` in line 84 and do nothing in line 86 (unifying the schedule_source type to str), This PR do nothing on line 84 and adds the `read()` in line 86.

Reply:
Applied the minor change of the `read` call in the same line as the PR in #120. This way I tested the code and approved the PR there. Either one, #99 or #120 which gets in first will fix the issue in the same way.